src/components/Update.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { View } from 'react-native';
+import { StyledText as Text, SubHeader } from '.';
+
+const Update = () => (
+  <View >
+    <SubHeader pageHeaderText="UPDATE" />
+    <Text>Hello</Text>
+  </View>
```

Comment:
What is this component for ma?

Reply:
Lol. I just updated the description of the PR now.

Reply:
Awesome. Seen!